### PR TITLE
Add envio

### PR DESCRIPTION
**Pull request by Jordy-Baby**

### CODE REVIEW

File: src/config/projects.ts [hunk 1]
```diff
@@ -175,6 +176,13 @@ export const projects = (t: TFunction): PortalMeta[] => [
     ),
     tags: ["Wallet"],
   },
+  logo: envio,
+  name: "Envio",
+    link: "https://envio.dev/",
+    description:
+      "Envio is a modern, multi-chain EVM blockchain indexing framework speed-optimized for querying real-time and historical data.",
+    tags: ["Infrastructure", "Tool"],
+  },
```
Comment (hackfisher) [inline src/config/projects.ts]:
```suggestion
  {
    logo: envio,
    name: t("Envio"),
    link: t("https://envio.dev/"),
    description: t(
      "Envio is a modern, multi-chain EVM blockchain indexing framework speed-optimized for querying real-time and historical data."
    ),
    tags: ["Infrastructure", "Tool"],
  },
```

**Review by hackfisher — CHANGES_REQUESTED** [review]

File: src/config/projects.ts [hunk 2]
```diff
@@ -48,6 +48,7 @@ import subApiDao from "../assets/projects/subapi-dao.png";
 import helixDao from "../assets/projects/helix-dao.png";
 import immunefi from "../assets/projects/immunefi.png";
 import ktonStaking from "../assets/projects/kton-staking.png";
+import envio from "..assets/projects/Envio_logo_sqaure.png";
```
Comment (hackfisher) [inline src/config/projects.ts]:
```suggestion
import envio from "../assets/projects/Envio_logo_sqaure.png";
```

**Review by hackfisher — CHANGES_REQUESTED** [review]

**Review by hackfisher — APPROVED** [review]

**Comment by nblogist** [comment]
@hackfisher as pointed out by @Jordy-Baby that Envio is not visible in the apps.darwinia.network - upon investigation we found out that the checks aren't completely passed and there is a vercel deployment issue: https://github.com/darwinia-network/apps/pull/413/checks can you please have a look at it? - in local deployments Envio shows up fine
image

**Comment by Jordy-Baby** [comment]
Hey, @hackfisher, I'm just following up on this. Thanks!

**Comment by fewensa** [comment]
fix #414 

Thanks for your work @Jordy-Baby  and @nblogist  now you can see it from https://apps.darwinia.network

**Pull request merged**